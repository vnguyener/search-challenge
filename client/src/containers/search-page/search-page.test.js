import React from 'react';
import { Provider } from 'react-redux';
import { configure } from 'enzyme';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import Adapter from 'enzyme-adapter-react-16';
import SearchPage from '.';

configure({ adapter: new Adapter() });
const mockStore = configureMockStore([thunk]);

describe('search page container', () => {
  it('renders correctly', () => {
    const store = mockStore({
      profiles: {
        profilesList: [],
      },
    });

    const wrapper = renderer
      .create(
        <Provider store={store}>
          <SearchPage />
        </Provider>
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
