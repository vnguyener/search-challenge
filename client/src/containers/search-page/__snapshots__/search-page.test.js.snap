// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search page container renders correctly 1`] = `
<main
  className="search-page-container"
>
  <div
    className="filter-container"
  >
    <button
      className="minimal-button"
      disabled={true}
      style={Object {}}
    >
      <img
        alt="filter"
        src="filter.svg"
        width={22}
      />
    </button>
    <button
      className="minimal-button"
      disabled={true}
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="Sort ascending"
        src="./ascending.svg"
        width={22}
      />
    </button>
    <button
      className="minimal-button"
      disabled={true}
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="Sort descending"
        src="./descending.svg"
        width={22}
      />
    </button>
  </div>
  <div
    className="profiles-container"
  />
  <p
    className="no-results"
  >
    We're sorry, but there are no results to show.
  </p>
</main>
`;
